### PR TITLE
Pick up the latest startup-script image

### PR DESCRIPTION
**Pull request by michi-covalent**
Upgrading this image is not automated yet.

Ref: #25773
Ref: https://github.com/cilium/image-tools/pull/218
Ref: https://quay.io/repository/cilium/startup-script?tab=tags

Signed-off-by: Michi Mutsuzaki <michi@isovalent.com>

### CODE REVIEW

**Comment by michi-covalent** [comment]
i think we can consider this a security relevant fix. i'll set backport labels accordingly.

File: install/kubernetes/Makefile.values [hunk 1]
```diff
@@ -38,7 +38,7 @@ export CILIUM_ETCD_OPERATOR_REPO:=quay.io/cilium/cilium-etcd-operator
 export CILIUM_ETCD_OPERATOR_VERSION:=v2.0.7
 export CILIUM_ETCD_OPERATOR_DIGEST:=sha256:04b8327f7f992693c2cb483b999041ed8f92efc8e14f2a5f3ab95574a65ea2dc
 export CILIUM_NODEINIT_REPO:=quay.io/cilium/startup-script
-export CILIUM_NODEINIT_VERSION:=d69851597ea019af980891a4628fb36b7880ec26
+export CILIUM_NODEINIT_VERSION:=5e928f628f9fc644a1d2651d6cd6aecbde0e7acd
```
Comment (sayboras) [inline install/kubernetes/Makefile.values]: This image is actually published 1 year ago, there is no harm to check in image-tools if there is any upgrade we need to do :thinking:

**Review by sayboras — APPROVED** [review]
One comment as per below

File: install/kubernetes/Makefile.values (hunk 1 above)
Comment (michi-covalent) [reply install/kubernetes/Makefile.values]: it is so tempting to click the merge button when it's green, but i stayed patient and opened https://github.com/cilium/image-tools/pull/218.

Comment (sayboras) [reply install/kubernetes/Makefile.values]: yup, it will be great if we can do it in one shot, you know, I am backporter this week :laughing:

Comment (michi-covalent) [reply install/kubernetes/Makefile.values]: alright let me convert this to draft, let's get https://github.com/cilium/image-tools/pull/218 merged first.

**Comment by michi-covalent** [comment]
[test-1.25-4.19](https://jenkins.cilium.io/job/Cilium-PR-K8s-1.25-kernel-4.19/293/): a transient failure downloading something from github. re-triggering...
```
Unable to download helm chart v1.13 from GitHub
Expected command: mkdir -p /home/jenkins/workspace/Cilium-PR-K8s-1.25-kernel-4.19/src/github.com/cilium/cilium/old-charts/v1.13 && cd /home/jenkins/workspace/Cilium-PR-K8s-1.25-kernel-4.19/src/github.com/cilium/cilium/old-charts/v1.13 && rm -rf * && wget https://github.com/cilium/cilium/archive/refs/heads/v1.13.tar.gz && tar -xf v1.13.tar.gz 
```

**Comment by michi-covalent** [comment]
/test-1.25-4.19

**Pull request merged**